recipes/cutadapt/meta.yaml

```diff
@@ -35,8 +38,12 @@ test:
     - cutadapt --version
 
 about:
-  home: https://cutadapt.readthedocs.io/
+  home: https://github.com/marcelm/cutadapt
```

Comment:
I do not approve. It was my intention to use the readthedocs page as the home page.